src/lib/OpenEXR/ImfCRgbaFile.h

```diff
@@ -68,18 +68,20 @@ typedef struct ImfRgba ImfRgba;
 
 /*
 ** Compression types; values must be the same as in Imf::Compression.
+** DO NOT EDIT: THIS FILE IS GENERATED BY CMAKE.
 */
 
-#define IMF_NO_COMPRESSION 0
-#define IMF_RLE_COMPRESSION 1
-#define IMF_ZIPS_COMPRESSION 2
-#define IMF_ZIP_COMPRESSION 3
-#define IMF_PIZ_COMPRESSION 4
-#define IMF_PXR24_COMPRESSION 5
-#define IMF_B44_COMPRESSION 6
-#define IMF_B44A_COMPRESSION 7
-#define IMF_DWAA_COMPRESSION 8
-#define IMF_DWAB_COMPRESSION 9
+#define IMF_NO_COMPRESSION = 0,
```

Comment:
Is this correct? Is the `=` allowed here? And does `IMF_NUM_COMPRESSION_METHODS` need a value?

Reply:
oops. yes it does !